tools/tertiary-analysis/seurat/seurat_export_cellbrowser.xml

```diff
@@ -0,0 +1,69 @@
+<tool id="seurat_export_cellbrowser" name="Seurat Export2CellBrowser" version="2.3.1+galaxy0">
+    <description>produces files for UCSC CellBrowser import.</description>
+    <macros>
+        <import>seurat_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <expand macro="version" />
+    <command detect_errors="exit_code"><![CDATA[
+#if $markers_file:
+  cp '$markers_file' markers.csv;
```

Comment:
yes, the issue here is that one of the processes copies everything into a new directory, and hence I wanted the actual file copied instead of the link (the file shouldn't be heavy anyway).